CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 2.8.1
+
+- implemented executing a prepared statement with question mark placeholder style similar to PHP PDO
+```dart
+ final results = await connection.query(
+        ' SELECT * FROM public.table_example LIMIT ?',
+        substitutionValues: [2000], placeholderIdentifier: 
+        PlaceholderIdentifier.onlyQuestionMark);
+```
+
+## 2.7.1
```

Comment:
It's unlikely that your PR will get released twice, so a single version header in the changelog should be enough :)